tfs/src/main/java/hudson/plugins/tfs/rm/ReleaseManagementCI.java

```diff
@@ -163,6 +241,36 @@ void CreateRelease(
             JSONObject object = new JSONObject(response);
             listener.getLogger().printf("Release Name: %s%n", object.getString("name"));
             listener.getLogger().printf("Release id: %s%n", object.getString("id"));
+
+            if (StringUtils.isNotBlank(this.destinationPath)) {
```

Comment:
also can we wait for release completion only if isArchiveLog is set to true. with the current change it always waits for release completion whether we archive or not. If someone does not want to wait for release completion, he could uncheck isArchiveLog.

Reply:
In the current change, if user leave `destination path` blank, we won't wait for release completion. Do you means that if user uncheck `isArchiveLog`, whether he fills in `destination path` or not, we won't download logs for he.